tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java

```diff
@@ -215,6 +225,33 @@ public void testSiteUseCasesForCreateModifyDeleteSite()
         assertSiteQueryResult(testSite1.getId(), List.of());
         assertSiteQueryResult(ALL_SITES, List.of());
         assertSiteQueryResult(EVERYTHING, "AND", FILE_CONTENT_CONDITION, List.of(fileNotInSite));
+
+        Step.STEP("Moving files between sites and modifying content use cases");
+
+        SiteModel publicSite1 = dataSite.usingUser(testUser).createSite(new SiteModel("public-site-1"));
+        SiteModel publicSite2 = dataSite.usingUser(testUser).createSite(new SiteModel("public-site-2"));
```

Comment:
there is a createSite method you could use if you wish to

Reply:
Sure, I'll replace it

Reply:
used as you suggested ;)